solution/ui/regulations/eregs-vite/src/views/Resources.vue

```diff
@@ -142,7 +138,7 @@ export default {
             },
             supplementalContent: [],
             searchInputValue: "",
-            URL: import.meta.env.VITE_API_URL
+            URL: import.meta.env.VITE_API_URL,
```

Comment:
I think this is actually no longer used.

Reply:
undid the change but merging didnt remove this